csrc/apis/java/mmdeploy/Classifier.java

```diff
@@ -0,0 +1,49 @@
+package mmdeploy;
+
+public class Classifier {
+    static {
+        System.loadLibrary("mmdeploy_java");
+    }
+
+    private final long handle;
+
+
+    public record Result(int label_id, float score) {
```

Comment:
No member variables?

Reply:
This is a [record class](https://www.baeldung.com/java-record-keyword). However it is available for JAVA 14 (released early 2020) and later, we may need to replace it with an class.

Reply:
I agree with replacing it with an ordinary class. According to [Java SE Support Roadmap](https://www.oracle.com/java/technologies/java-se-support-roadmap.html),  `JAVA 14` is not the LTS version. According to this [poster](https://www.stackchief.com/blog/Which%20Version%20of%20Java%20Should%20You%20Use%3F), "even in 2022, many applications continue to run on Java 8 and Java 11." 
So let's change `record` to a common Java class.